db/migrations/mysql/000059_upgrade_users_v6.0.down.sql

```diff
@@ -36,16 +36,37 @@ SET @preparedStatement = (SELECT IF(
         WHERE table_name = 'Users'
         AND table_schema = DATABASE()
         AND column_name = 'Timezone'
-        AND column_default = NULL
+        AND column_type = 'JSON'
```

Comment:
Isn't this same as the previous one? `!= 'varchar(256)'` would mean anything right?

Reply:
Hmm, actually we can use either. 0/5 checking if data type is `JSON`. This is for purely reverting the `up` script. Actually both would do the same in this context. And there are two prepared statements for this column; one for column_type the other one is column_default.

Reply:
Ok, let's add a comment if we are keeping it unchanged. :+1:

Reply:
Oh man, I'm removing this. I need more coffee :) You're right the previous statement is enough.

Reply:
No worries man. You need a break. :)